processing/src/test/java/io/druid/segment/data/BenchmarkIndexibleWrites.java

```diff
@@ -171,7 +171,7 @@ private void ensureCapacity(int capacity)
   private final Integer totalIndexSize = 1<<20;
 
   @BenchmarkOptions(warmupRounds = 100, benchmarkRounds = 100, clock = Clock.REAL_TIME, callgc = true)
-  @Test
+  //@Test
```

Comment:
IIRC if you have the test class extend extends `com.carrotsearch.junitbenchmarks.AbstractBenchmark` then it doesn't run in the general maven workflow, and instead must be explicitly requested.